linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/tests/chrony_d_one_pool_misconfigured.fail.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# packages = chrony
+
+{{{ bash_package_remove("ntp") }}}
+
+# Remove all server or pool options
+sed -i "/^\(server\|pool\).*/d" {{{ chrony_d_path }}}/20-pools.conf
```

Comment:
These configs seem platform specific. Maybe the tests should be made platform specific?

Reply:
I tend to agree with the above comment, instead of making it platform specific we can replace `20-pools.conf` with wildcard